scripts/run_js_tests.sh

```diff
@@ -40,7 +40,9 @@ source $(dirname $0)/setup_gae.sh || exit 1
 
 # Install third party dependencies
 # TODO(sll): Make this work with fewer third-party dependencies.
-bash scripts/install_third_party.sh
+# Travis aborts test run if nothing is printed back to STDOUT for some time.
+# -x is used to avoid that.
+bash -x scripts/install_third_party.sh
```

Comment:
Is it possible to move the -x into the run_js_tests.sh invocation in travis.yml (on line 39), so that running this script standalone doesn't produce a large debug log?